contracts/ActivePool.sol

```diff
@@ -80,6 +84,12 @@ contract ActivePool is
 		renounceOwnership();
 	}
 
+	function setCollStakingManagerAddress(address _collStakingManagerAddress) external {
```

Comment:
We need to set it owner function only. Since ActivePool's owner is revoked, either create a onetime funciton to set the owner to the Admin MS. Or create a second ownership on it.